messaging/MessagingExampleSwift/AppDelegate.swift

```diff
@@ -134,7 +134,7 @@ extension AppDelegate : UNUserNotificationCenterDelegate {
     print(userInfo)
 
     // Change this to your preferred presentation option
-    completionHandler([])
+    completionHandler([[.alert, .sound]])
```

Comment:
Looks like the Objective-C sample has the same issue?

Reply:
done.